src/content/docs/en/concepts/why-astro.mdx

```diff
@@ -1,45 +1,63 @@
 ---
 title: Why Astro?
-description: "Astro is an all-in-one web framework for building fast, content-focused websites. Learn more."
+description: "Astro is an all-in-one web framework for building fast, content-driven websites. Learn more."
 i18nReady: true
 ---
 
-Astro is an **all-in-one** **web framework** for building **fast,** **content-focused websites.**
+**Astro** is a web framework for building **content-driven websites** that load fast with great SEO.
 
-Why choose Astro over another web framework? Here are five core design principles to help explain why we built Astro, the problems that it exists to solve, and why Astro may be the best choice for your project or team.
+:::tip
+Explore the [collection of Astro themes](https://astro.build/themes/) for blogs, portfolios, docs, landing pages, SaaS, marketing, ecommerce sites, and more!
+::: 
+
+Astro is best known for pioneering and popularizing [**islands**](/en/concepts/islands/): a JavaScript frontend architecture optimized for performance, flexibility, and simplicity.
+
+## Features
+
+Astro includes everything you need built-in to get started quickly, with hundreds of available integrations for added features, performance, and developer experience. 
+
+Some highlights include:
 
-#### Astro is...
+- **[Islands:](/en/concepts/islands/)** A component-based web architecture optimized for content-driven websites.
+- **[UI-agnostic](/en/core-concepts/framework-components/):** Supports React, Preact, Svelte, Vue, Solid, Lit, HTMX, web components, and more.
+- **[Server-first:](/en/core-concepts/rendering-modes/)** Moves expensive rendering off of your visitors' devices.
+- **[Zero JS, by default:](/en/core-concepts/astro-components/)** Less client-side JavaScript to slow your site down.
+- **[Content collections:](/en/guides/content-collections/)** Organize, validate and provide TypeScript type-safety for your Markdown content.
+- **[Customizable:](/en/guides/integrations-guide/)** Tailwind, MDX, and hundreds of integrations to choose from.
 
-1. [Content-focused](#content-focused): Astro was designed for content-rich websites.
+## Design Principles
+
+Here are five core design principles to help explain why we built Astro, the problems that it exists to solve, and why Astro may be the best choice for your project or team.
+
+Astro is...
+
+1. [Content-driven](#content-driven): Astro was designed to showcase your content.
 2. [Server-first](#server-first): Websites run faster when they render HTML on the server.
 3. [Fast by default](#fast-by-default): It should be impossible to build a slow website in Astro.
 4. [Easy to use](#easy-to-use): You don't need to be an expert to build something with Astro.
-5. [Fully-featured, but flexible](#fully-featured-but-flexible): Over 100+ Astro integrations to choose from.
+5. [Developer-focused](#developer-focused): You should have the resources you need to be successful.
```

Comment:
Since each bulleted list item's "title" is bold just above, maybe we should use the same formatting here for consistency?